### PR TITLE
feat(project-information): add stage progress tag and refactor projec…

### PR DESCRIPTION
**Pull request by emil-placheta**
…t stages

### CODE REVIEW

File: packages/forms-web-app/src/pages/projects/project-information/controller.js [hunk 1]
```diff
@@ -13,6 +13,7 @@ const getProjectInformation = async (req, res, next) => {
 
 		const projectUpdates = await getProjectUpdatesData(caseRef);
 
+		console.log('APPDATA :>> ', applicationData);
```
Comment (emil-placheta) [inline packages/forms-web-app/src/pages/projects/project-information/controller.js:16]: remove

File: packages/forms-web-app/src/pages/projects/project-information/controller.test.js [hunk 2]
```diff
@@ -12,7 +12,7 @@ const commonMockData = {
 	Proposal: 'I am the proposal',
 	Summary: 'I am the project summary data',
 	WebAddress: 'mock-web-address',
-	dateOfNonAcceptance: '2020-01-01',
```
Comment (emil-placheta) [inline packages/forms-web-app/src/pages/projects/project-information/controller.test.js:15]: undo

File: packages/forms-web-app/src/pages/projects/project-information/controller.test.js [hunk 3]
```diff
@@ -12,7 +12,7 @@ const commonMockData = {
...
+	DateOfNonAcceptance: '2020-01-01',
```
Comment (emil-placheta) [inline packages/forms-web-app/src/pages/projects/project-information/controller.test.js:15]: undo

**Review by jonathangoulding — APPROVED** [review]

**Pull request merged**